scripts/qa_crowbarsetup.sh

```diff
@@ -2536,6 +2537,16 @@ function oncontroller_testsetup()
     . .openrc
 
     export LC_ALL=C
+
+    # prepare test image with the -test packages containing functional tests
+    if iscloudver 6plus; then
+        local mount_dir="/tmp/Cloud-Testing"
```

Comment:
right, fine with me. I just don't want to mount anything under /tmp. can we put it under /var/lib/Cloud-Testing ?